Runtime/Weapon/CWaveBuster.cpp

```diff
@@ -2,7 +2,15 @@
 
 #include "Runtime/CSimplePool.hpp"
 #include "Runtime/GameGlobalObjects.hpp"
+#include "Runtime/CStateManager.hpp"
```

Comment:
These should probably be alphabetically sorted.

Reply:
I'll fix that when I'm done implementing CIceSheegoth.